floc.bs

```diff
@@ -1 +1,169 @@
+
+  <h3 id="sensitive-information">Sensitive information</h3>
+  An [=interest cohort=] might reveal sensitive information. As a first mitigation, the browser should remove sensitive categories from its data collection. But this does not mean sensitive information can’t be leaked. Some people are sensitive to categories that others are not, and there is no globally accepted notion of sensitive categories.
+
+  Cohorts could be evaluated for fairness by measuring and limiting their deviation from population-level demographics with respect to the prevalence of sensitive categories, to prevent their use as proxies for a sensitive category. However, this evaluation would require knowing how many individual people in each cohort were in the sensitive categories, information which could be difficult or intrusive to obtain.
+
+  It should be clear that FLoC will never be able to prevent all misuse. There will be categories that are sensitive in contexts that weren't predicted. Beyond FLoC's technical means of preventing abuse, sites that use cohorts will need to ensure that people are treated fairly, just as they must with algorithmic decisions made based on any other data today.
+
+  <h3 id="tracking-people-via-their-interest-cohort">Tracking people via their interest cohort</h3>
+  An [=interest cohort=] could be used as a user identifier. It may not have enough bits of information to individually identify someone, but in combination with other information (such as an IP address), it might. One design mitigation is to ensure cohort sizes are large enough that they are not useful for tracking. In addition, if the user agent believes that too many high-entropy bits of information have already been consumed by a given {{Document}}, then the {{Document/interestCohort()}} algorithm will return a rejected promise, which can help mitigate such tracking.
+
+  The time series of [=interest cohorts=] can also be used as a user identifier. Sites could associate users' first-party identity with a series of [=interest cohorts=] observed over time, and could report these series to a single tracking service. The tracking service could then associate each series with the sites to know the browsing history of an individual.
+
+  One design mitigation is to compute the [=interest cohort=] on the fly so that sites are less likely to observe the same [=interest cohort=] for a given user. However, one could imagine that those unstable [=interest cohorts=], when casting to a lower dimension, are still going to be stable, and the above concerns still apply though to a lesser extent.
+
+  <h3 id="recovering-the-browsing-history-from-cohorts">Recovering the browsing history from cohorts</h3>
+  Updating the [=interest cohort=] too often may let a given site learn too much about the user. An attacker could attempt to use compressed sensing to recover the browsing history.
```

Comment:
What is "compressed sensing"? A hyperlink to a paper or Wikipedia article or something would be helpful.

Reply:
Done.